api/Parser/Commit.cs

```diff
@@ -0,0 +1,10 @@
+namespace api.Parser;
+
+public class Commit
```

Comment:
Dit niet beter verplaatsen naar `Model/Parser` ?

Reply:
en renamen naar `GitCommit`

Reply:
Nee is geen `GitCommit`, het is een entry specifiek voor het parsen, en zit daarom ook in die namespace